lib/inherited_proofing/logingov/service.rb

```diff
@@ -14,7 +14,8 @@ class Service < Common::Client::Base
       # Login.gov will pass this param back for validation in their user attributes call
       def render_auth(auth_code:)
         renderer = ActionController::Base.renderer
-        renderer.controller.prepend_view_path(Rails.root.join('lib', 'inherited_proofing', 'logingov', 'templates'))
```

Comment:
this should probably stay the string since the constant would change the filepath